src/main/java/mekanism/common/integration/multipart/MultipartTransmitter.java

```diff
@@ -28,6 +30,28 @@ public IPartSlot getSlotFromWorld(IBlockAccess world, BlockPos pos, IBlockState
 	}
 	
 	@Override
+	public void onAdded(IPartInfo part)
+	{
+		TileEntity tile = part.getTile().getTileEntity();
+		
+		if(tile instanceof TileEntitySidedPipe) 
+		{
+			((TileEntitySidedPipe)tile).onPartChanged(null);
+		}
+	}
+	
+	@Override
+	public void onPartAdded(IPartInfo part, IPartInfo otherPart)
+	{
+		TileEntity tile = part.getTile().getTileEntity();
+		
+		if(tile instanceof TileEntitySidedPipe) 
+		{
+			tile.validate();
```

Comment:
In this case MCMP validates the new part being added (in this case the chiseled block) but before it does that it ends up changing the block state at the position of the transmitter which causes the block to break thereby invalidating the transmitter. Since the transmitter TileEntity should still be valid (it just exists as part of the new multipart block) this works around that.